src/python/setup.py

```diff
@@ -32,15 +30,27 @@
     author="The tensorflow cloud authors",
     author_email="tensorflow-cloud@google.com",
     license="Apache License 2.0",
-    install_requires=["docker", "google-api-python-client", "google-cloud-storage",],
-    extras_require={"tests": ["pytest", "flake8", "mock"],},
+    extras_require={"tests": dependencies.make_required_test_packages()},
+    include_package_data=True,
+    install_requires=dependencies.make_required_install_packages(),
     classifiers=[
-        "Programming Language :: Python",
+        "Development Status :: 4 - Beta",
         "Operating System :: Unix",
         "Operating System :: MacOS",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Education",
         "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3",
```

Comment:
Why do we need to specify Python versions?

Reply:
Common practice to include for PyPI package information. Tensorflow also follows this, see [tf setup.py](https://github.com/tensorflow/tensorflow/blob/82e073b3a891ddce4136dc106d5fe8f0cba75389/tensorflow/tools/pip_package/setup.py#L287)